artemis-commons/src/main/java/org/apache/activemq/artemis/utils/actors/ProcessorBase.java

```diff
@@ -117,13 +112,6 @@ public int shutdownNow(Consumer<? super T> onPendingItem, int timeout, TimeUnit
       requestedShutdown = true;
       yielded = false;
 
-      if (!inHandler()) {
-         // We don't have an option where we could do an immediate timeout
-         // I just need to make one roundtrip to make sure there's no pending tasks on the loop
-         // for that I ellected one second
-         flush(timeout, unit);
```

Comment:
This was for tests and it was still causing issues anyway...


so, time to go !!! more issues than benefits